docs/source/models/gbd2017/causes/neural_tube_defects/index.rst

```diff
@@ -114,6 +140,16 @@ Data Description
 
    Add tables describing data sources for the Vivarium model.
 
+
+Restrictions
+++++++++++++
+
+yll_age_group_id_start=2,
+yll_age_group_id_end=17,
+
+yld_age_group_id_start=2,
+yld_age_group_id_end=235,
```

Comment:
Let's say in words what these age groups mean. IIUC, `2` is "early neonatal".  Does `235` mean "birth prevalence"?  And what is `17`?

Reply:
I need to look these up.  I moved this into an explicit to-do in the next pull request (https://github.com/ihmeuw/vivarium_research/pull/55).  At the doc session today we discussed how to format the restrictions table, including listing actual ages rather than ids, so I'll incorporate that when I fill in the Restrictions section.